en/components/grid/selection_based_aggregates.md

```diff
@@ -0,0 +1,66 @@
+### Selection-based Aggregates
```

Comment:
Metadata is missing. You also need to add the same markdown files for JP and KR for localization.

Reply:
Done